app/controllers/cypress/factories_controller.rb

```diff
@@ -46,5 +73,24 @@ def params_to_attributes(params)
 
         return attributes, should_validate
       end
+
+      def transform_hash(value)
```

Comment:
We should keep that as it makes sure that arrays are parsed correctly. Without it, something like
```
FactoryBot.create("tutorial", { lecture_id: 1, tutor_ids: [2] })
```
lands in Rails as
```
[:tutorial, {"lecture_id"=>"1", "tutor_ids"=>{"0"=>"2"}}]
```
whereas what we need (and what is made sure of by `transform_hash`) is
```
[:tutorial, {"lecture_id"=>"1", "tutor_ids"=>["2"]}]
```
If you have a good idea for a docstring please add it.
`transform_hash_keys` (in l79-81) replaces hashes all of whose keys are integers by the array of their values.